fsw/src/ds_file.c

```diff
@@ -1025,6 +1031,31 @@ void DS_FileTestAge(uint32 ElapsedSeconds)
 
 } /* End of DS_FileTestAge() */
 
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+/*                                                                 */
+/* DS_FileTransmit() - transmit file info                          */
+/*                                                                 */
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+
+void DS_FileTransmit(DS_AppFileStatus_t *FileStatus)
+{
+    DS_FileCompletePkt_t DS_FileCompletePkt = {.FileStatus = *FileStatus};
```

Comment:
Recommend using the FileInfo structure in the packet and copy elements from FileStatus into FileInfo.